client/service/src/client/call_executor.rs

```diff
@@ -253,7 +253,8 @@ where
 				)
 				.set_parent_hash(at_hash);
 				// TODO: https://github.com/paritytech/substrate/issues/4455
-				// .with_storage_transaction_cache(storage_transaction_cache.as_mut().map(|c| &mut **c))
+				// .with_storage_transaction_cache(storage_transaction_cache.as_mut().map(|c| &mut
```

Comment:
a bit strange now, but this comment is kinda invalid anyway. The example code should be in the issue.